src/custom-apps/queries.ts

```diff
@@ -7,3 +7,17 @@ export const webhooksDetails = gql`
     }
   }
 `;
+
+// export const introspectionQueryList = gql`
+//   query IntrospectionQueryList {
+//     __schema {
+//       queryType {
+//         name
+//         fields {
+//           name
+//           description
+//         }
+//       }
+//     }
+//   }
+// `;
```

Comment:
**todo:**
If codegen can't handle `__schema` please add here a comment explaining this and point to query in utils file.

Reply:
@Droniu done